meet-data/ipf/ipf-parse-post

```diff
@@ -7,13 +7,40 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 REPOSCRIPTDIR="${SCRIPTDIR}/../../scripts"
 REPOTESTDIR="${SCRIPTDIR}/../../tests"
 
-# Use IPF-specific knowledge to parse results.csv into entries.csv.
-${SCRIPTDIR}/ipf-standardize-csv results.csv > entries.csv
+if [ -e results.csv ];then
+	# Use IPF-specific knowledge to parse results.csv into entries.csv.
+	${SCRIPTDIR}/ipf-standardize-csv results.csv > entries.csv
+else
+	n=`ls  | grep 'results.*\.csv' | wc -l`
+
+	for (( ii=1; ii<=${n};ii++))
+	do
+		${SCRIPTDIR}/ipf-standardize-csv "results"$ii".csv" > "entries"$ii".csv"
```

Comment:
If this is using `${SCRIPTDIR}`, then you shouldn't need the `csv-cat` symlink.

Reply:
Good point, removed this symlink.